docassemble/juvenilesealing/data/questions/juvenile-sealing-text-10.yml

```diff
@@ -208,7 +184,7 @@ subquestion: |
   0 Jump St., Apt. # 2[BR]
   Boston, MA 02114
 
-continue button label: Keep going ${next_icon}
+continue button label: Continue ${next_icon}
```

Comment:
I was thinking 'Keep going' or 'Next' might be words that are more plain than 'Continue'. Thoughts?